src/themes/material/assets/cell.edit.scss

```diff
@@ -123,9 +123,30 @@ td.q-grid-edit {
 		width: 100%;
```

Comment:
please if you need to change something for mat, add new file with "mat" prefix(mat.cell.edit in this case)

Reply:
Done